### ISSUE TITLE
"as any" exception to no-object-literal-type-assertion

### ISSUE THREAD

**Issue by KurtPreston** [palantir/tslint#2716]
I really like the `no-object-literal-type-assertion` rule for cases where I'm trying to type a variable.  i.e., this is great:

```typescript
// What I should write:
const x: MyInterface = {...}

// What tslint wisely stops me from writing:
const x = {...} as MyInterface
```

However, the one circumstance where I tend to use the `as` syntax is specifically when doing type overrides for tests.  i.e.

```typescript
// What I want to write for my test:
const x = {...} as any

// What tslint recommends:
const x: any = {...}
```

In this case, I'm using `as any` to draw attention to the fact that it *is* an override... the variable `x` is not actually an `any` type. 

Would it be possible to add an option to ignore `no-object-literal-type-assertion` for `as any`?

**Comment by ajafff** [comment]
That was already done in a recent PR and will be part of the next release.

Am 08.05.2017 21:21 schrieb "Kurt Preston" <notifications@github.com>:

**Comment by adidahiya** [comment]
fixed by #2671
